codebuild/osx.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set +ex
+
+OS=$(uname -s)
+CPP_VER="1.7.163"  #github versioned branch
+BUILDROOT="/var/tmp/build-$(date +%s)"
+OPENSSLVER="openssl@1.1" #note this is a brew label, not an exact version reference
```

Comment:
NP: maintain same variable style `OPENSSL_VER`

Reply:
addressed in next commit